### PR TITLE
feat: change input pointcloud from outside of launch file

### PR DESCRIPTION
**Pull request by taichiH**
## PR Type

- Improvement

## Related Links
tier4/aip_launcher#24

## Description
I've changed input topic of localization and perception module from rectified/pointcloud to outlier_filtered/pointcloud.
It was not possible to pass a topic name to lidar detection node from launcher as an argument so far, so I fixed that as well.

@h-ohta added
- make `container` as argument
- ~~rearrange output topic arg~~ <- sipin out to #270 

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Code follows [coding guidelines][coding-guidelines]
- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR**: Required to pass before the merge.
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: localization_launch/launch/util/util.launch.py [hunk 1]
```diff
@@ -106,30 +116,13 @@ def add_launch_arg(name: str, default_value=None, description=None):
         "path to the parameter file of random_downsample_filter",
     )
     add_launch_arg("use_intra_process", "true", "use ROS2 component container communication")
+    add_launch_arg("use_pointcloud_container", "False")
+    add_launch_arg("container_name", "perception_pipeline_container")
```
Comment (YamatoAndo) [inline localization_launch/launch/util/util.launch.py]: why `perception_pipeline_container`?
I think `pointcloud_container` is better.

Comment (h-ohta) [reply localization_launch/launch/util/util.launch.py]: Sorry, It is my mistake...

**Comment by yukkysaito** [comment]
Since it includes the issue of what to do with the container, I will separate PR and import the modified topic name only.
https://github.com/autowarefoundation/autoware.universe/pull/645

**Comment by yukkysaito** [comment]
@h-ohta 
LGTM and add small modifications.
Can you review this PR?

File: localization_launch/launch/localization.launch.xml [hunk 2]
```diff
@@ -1,12 +1,17 @@
 <?xml version="1.0"?>
 <launch>
+  <arg name="input/pointcloud" default="/sensing/lidar/top/rectified/pointcloud" description="The topic will be used in the localization util module"/>
```
Comment (h-ohta) [inline localization_launch/launch/localization.launch.xml:3]: @yukkysaito What is the reason to restore this topic name?
This PR is to changing rectified to outlier.

Comment (yukkysaito) [reply localization_launch/launch/localization.launch.xml:3]: The topic name was not changed this time.
image

Comment (h-ohta) [reply localization_launch/launch/localization.launch.xml:3]: It seems arguments are deleted…?

Comment (yukkysaito) [reply localization_launch/launch/localization.launch.xml:3]: @h-ohta 
image

Comment (h-ohta) [reply localization_launch/launch/localization.launch.xml:3]: I'm sorry I can't understand. Could you explain a bit more by words?

Comment (yukkysaito) [reply localization_launch/launch/localization.launch.xml:3]: sorry, I thought I had written a sentence. :pray: 
It seems input/pointcloud can be used as an argument.

Comment (h-ohta) [reply localization_launch/launch/localization.launch.xml:3]: Thank you, I see. 
I'm sorry I think my first question made confuse.

My question is that "why you use /sensing/lidar/top/rectified/pointcloud again?".
This PR title is "change input topic of localization ~ to outlier_filtered/pointcloud". 
So I believe that "/sensing/lidar/top/outlier/pointcloud" can be used here.

Comment (h-ohta) [reply localization_launch/launch/localization.launch.xml:3]: Or, should we change PR titile?

Comment (yukkysaito) [reply localization_launch/launch/localization.launch.xml:3]: I see. i'll change title

Comment (yukkysaito) [reply localization_launch/launch/localization.launch.xml:3]: @h-ohta done

Comment (h-ohta) [reply localization_launch/launch/localization.launch.xml:3]: @yukkysaito That PR title is suspicious. The rectified/pointcloud topic has been already used before.
I think This PR is doing `change input pointcloud from outside of launch file`

Comment (yukkysaito) [reply localization_launch/launch/localization.launch.xml:3]: @h-ohta 
I fixed it
If there is anything you think of, you can correct it.

**Review by h-ohta — APPROVED** [review]

**Pull request merged**